doc/source/whatsnew/v0.21.0.txt

```diff
@@ -117,6 +117,7 @@ Other Enhancements
 - :func:`MultiIndex.is_monotonic_decreasing` has been implemented.  Previously returned ``False`` in all cases. (:issue:`16554`)
 - :func:`Categorical.rename_categories` now accepts a dict-like argument as `new_categories` and only updates the categories found in that dict. (:issue:`17336`)
 - :func:`read_excel` raises ``ImportError`` with a better message if ``xlrd`` is not installed. (:issue:`17613`)
+- :func:`assign` will preserve the original order of **kwargs for Python 3.6+ users 
```

Comment:
This will be

    :meth:`DataFrame.assign`

And put `**kwargs` in double backticks. And maybe reference the original behavior like "instead of sorting the column names".